plugins/processors/dedup/dedup.go

```diff
@@ -3,6 +3,9 @@ package dedup
 
 import (
 	_ "embed"
+	"fmt"
+	"github.com/influxdata/telegraf/plugins/parsers/influx"
+	influxSerializer "github.com/influxdata/telegraf/plugins/serializers/influx"
```

Comment:
Same here, keep telegraf imports together.

Maybe also rename the first one to `influxParser` to have no confusion at all in the code?